tests/mocha/astnode_test.js

```diff
@@ -7,10 +7,11 @@
 goog.module('Blockly.test.astNode');
 
 const {ASTNode} = goog.require('Blockly.ASTNode');
-const {sharedTestSetup, sharedTestTeardown, workspaceTeardown} = goog.require('Blockly.test.helpers');
+const {sharedTestSetup, sharedTestTeardown, workspaceTeardown} = goog.require('Blockly.test.helpers.setupTeardown');
 
 
 suite('ASTNode', function() {
+  console.log('1/a');
```

Comment:
Was this debugging that got committed by accident?  (Here and several places below).